spec/motion-spec/should_spec.rb

```diff
@@ -1,34 +1,31 @@
 # -*- encoding : utf-8 -*-
-motion_require '../spec_helper'
 
 describe "#should shortcut for #it('should')" do
-
-  should "be called" do
+  should 'be called' do
     @called = true
     @called.should.eq true
   end
 
-  should "save some characters by typing should" do
-    lambda { should.satisfy { 1 == 1 } }.should.not.raise
+  should 'save some characters by typing should' do
+    proc { should.satisfy { 1 == 1 } }.should.not.raise
   end
 
-  should "save characters even on failure" do
-    lambda { should.satisfy { 1 == 2 } }.should.raise Motion::Spec::Error
+  should 'save characters even on failure' do
+    proc { should.satisfy { 1 == 2 } }.should.raise MotionSpec::Error
   end
 
-  should "work nested" do
+  should 'work nested' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.